README.md

```diff
@@ -8,7 +8,7 @@ An HTTP server for Node designed to serve [PRPL](https://developers.google.com/w
 
 ### As a binary
 ```sh
-$ npm install -g prpl-server
```

Comment:
the problem with this is that yarn doesn't play well with nvm. It's given me some serious pause on recommending yar by default.